scalajslib/src/mill/scalajslib/ScalaJSWorkerApi.scala

```diff
@@ -0,0 +1,72 @@
+package mill.scalajslib
+
+import java.io.File
+
+import mill.api.{Ctx, Result}
+import mill.define.Discover
+import mill.scalajslib.api._
+import mill.{Agg, T}
+
+@deprecated("Use mill.scalajslib.worker.ScalaJSWorker instead", since = "mill 0.10.4")
+class ScalaJSWorker extends AutoCloseable {
+  private[scalajslib] val newWorker = new worker.ScalaJSWorker()
```

Comment:
This name is unfortunate.

Reply:
Yeah, I don't like it either. I want to implement the deprecated worker using the methods of the new worker.
Do you have any suggestion?

Reply:
I'll open a dedicated PR to your branch

Reply:
* https://github.com/lolgab/mill/pull/1